CryptoTechReminderSystem.AcceptanceTest/CryptoTechReminderSystemTests.cs

```diff
@@ -6,15 +6,15 @@
 using CryptoTechReminderSystem.UseCase;
 using FluentSim;
 using Newtonsoft.Json;
-using static CryptoTechReminderSystem.Gateway.MessageSender;
+using static CryptoTechReminderSystem.Gateway.SlackGateway;
 using static Newtonsoft.Json.JsonConvert;
 
 namespace CryptoTechReminderSystem.AcceptanceTest
 {
     public class CryptoTechReminderSystemTests
     {
         private FluentSimulator _fluentSimulator;
-        private MessageSender _messageSender;
+        private SlackGateway _slackGateway;
```

Comment:
:eyes: This should probably be the interface, not the concrete implementation. This would let you swap out the concrete implementation in the acceptance test without needing to update the type of the field

Reply:
Yes, you're right. It is now the interface. Cheers!